astro.config.mjs

```diff
@@ -136,6 +136,10 @@ export default defineConfig({
           label: 'Status',
           link: '/status',
         },
+        {
+          label: 'Security',
+          link: '/security',
```

Comment:
Is this PR meant to replace the one for shorebird.dev/security? Or are the two meant to co-exist? If so what's the difference in the content?